### ISSUE TITLE
@semantic-release/npm is being loaded by default

### ISSUE THREAD

**Issue by JayaKrishnaNamburu** [semantic-release/npm#151]
Hi,

I am trying to set up semantic release to my repo but I don't want to push to npm after success, so I removed ``@semantic-release/npm`` from my package.json but when I do a dry run a and a run in CI environment ``@semantic-release/npm`` is being loaded and packaged is being published to npm

<img width="629" alt="Screenshot 2019-03-13 at 12 43 54 PM" src="https://user-images.githubusercontent.com/11075561/54260106-013da300-458e-11e9-81b5-1fc82b46bc97.png">

<img width="926" alt="Screenshot 2019-03-13 at 12 43 34 PM" src="https://user-images.githubusercontent.com/11075561/54260126-0e5a9200-458e-11e9-8e4a-d355bc171e12.png">

**Comment by JayaKrishnaNamburu** [comment]
Fixed this by adding ``explicitly`` like this without npm
~~~
"publish": ["@semantic-release/github"]
~~~

but not sure why the ```npmPublish`` flag is reflecting when the publish is not explicitly defined.

~~~
[
  "@semantic-release/npm",
     {
         "npmPublish": false
     }
 ]
~~~

**Comment by pvdlg** [comment]
This question is answered in our FAQ: https://github.com/semantic-release/semantic-release/blob/master/docs/support/FAQ.md#can-i-skip-the-release-to-the-npm-registry

**Comment by JayaKrishnaNamburu** [comment]
> This question is answered in our FAQ: https://github.com/semantic-release/semantic-release/blob/master/docs/support/FAQ.md#can-i-skip-the-release-to-the-npm-registry

Hi @pvdlg, yes the FAQ says when ``npmPublish`` flag is set to false it will not publish to npm but that's not working for me, here are the additional details

~~~
"plugins": [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    "@semantic-release/changelog",
    [
      "@semantic-release/npm",
      {
        "npmPublish": false
      }
    ],
    "@semantic-release/github"
  ],
~~~

~~~
[6:19:24 AM] [semantic-release] [@semantic-release/npm] › ℹ  Published text-highlighter-regex@0.0.0-semantically-released on https://registry.npmjs.org/
[6:19:24 AM] [semantic-release] › ✔  Completed step "publish" of plugin "@semantic-release/npm"
~~~

Here is the [commit](https://github.com/JayaKrishnaNamburu/highlighter/commit/9ffc9c6972782a23d38fbba6b696952e84a0f902)

**Comment by JayaKrishnaNamburu** [comment]
> Fixed this by adding `explicitly` like this without npm
> 
> ```
> "publish": ["@semantic-release/github"]
> ```
> but not sure why the `npmPublish` flag is reflecting when the publish is not explicitly defined, and still need to define `NPM_TOKEN` even if it was being used.
> 
> ```
> [
>   "@semantic-release/npm",
>      {
>          "npmPublish": false
>      }
>  ]
> ```

When i explicitly define only ``github`` in the publish step only then it is skipping the process of publishing to npm. Do the docs need some update ? because if so i will be happy to update them and the ``assets`` is also not reflecting when it is configured as it is defined in docs --> https://github.com/semantic-release/github#usage 🤔
